vars/publishToNpm.groovy

```diff
@@ -45,3 +46,10 @@ void parameterCheck(String publicationType, String artifactPath) {
         error('publicationType: github does take any argument with it.')
     }
 }
+
+String getNpmTag(String githubTag) {
+    def matcher = githubTag =~ /-(\w+)\./
```

Comment:
Could you explain what exactly are you trying to match with examples for beta tag and latest tag?

Thanks.

Reply:
Anything after `-` in the version number. package.json has a version syntax of `1.0.0-beta.1` or `1.0.0`
So just grabbing the same. If there is nothing after major.minor.patch, it defaults to latest